### PR TITLE
JBTIS-451 - add JBDSIS 8.0.2.GA blog + images

### PR DESCRIPTION
**Pull request by pleacu**
JBTIS 4.2.2.Final/ JBDSIS 8.0.2.GA blog

### CODE REVIEW

File: blog/JBTIS-422-Final.adoc [hunk 1]
```diff
@@ -0,0 +1,169 @@
+= JBoss Tools Integration Stack - Integration tooling for Fuse, BRMS, and Data Virt + Early Access Data Transformation, SAP tooling and SwitchYard
```
Comment (maxandersen) [inline blog/JBTIS-422-Final.adoc]: okey - your titles are getting longer and longer breaking all kind of formatting when aggregated ;)

Can you please come up with a *smaller* title ? like "Integration Stack for Kepler now with Fuse and SAP support" or similar/shorter :)

File: blog/JBTIS-422-Final.adoc [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+:page-layout: blog
+:page-author: pleacu
+:page-date: 2015-06-29
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+
+*JBoss Fuse Tooling has arrived! - Plus an updated Teiid Designer and Early Access versions of the Fuse SAP Tool Suite, Fuse Data Transformation tooling and SwitchYard (Luna/ JBDS 8).*
```
Comment (maxandersen) [inline blog/JBTIS-422-Final.adoc]: no need to make this bold imo. just makes aggregation look weird.

File: blog/JBTIS-422-Final.adoc (hunk 1 above)
Comment (lhein) [inline blog/JBTIS-422-Final.adoc]: Kepler?

File: blog/JBTIS-422-Final.adoc [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+==== SwitchYard Highlights
+
+* link:https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Developer_Studio_Integration_Stack/8.0/html/8.0.2_Release_Notes/SwitchYard_2.0.0.Final.html[SwitchYard 2.0.0.Final Release Notes]
+
+==== The JBoss Tools website features tab
+
+Don't miss the link:/features[Features tab] for up to date information on your favorite Integration Stack components.
+
+== Installation
```
Comment (maxandersen) [inline blog/JBTIS-422-Final.adoc]: is all this about installation not the same for all blog entries ? how about move that to https://github.com/jbosstools/jbosstools-website/blob/master/downloads/installation.adoc instead and link to it ? (or just leave the simplest install option here (use eclipse marketplace) and link for more advanced details)

File: blog/JBTIS-422-Final.adoc (hunk 1 above)
Comment (maxandersen) [inline blog/JBTIS-422-Final.adoc]: sorry - Luna ;)

**Comment by pleacu** [comment]
creating a new PR to avoid confusion

**Pull request closed without merging**